### ISSUE TITLE
Periodic ouput for non interactive usage

### ISSUE THREAD

**Issue by jrob** [rsalmei/alive-progress#118]
I have a script that is going to have to run non interactively and is going to take a couple of days. I would like to see the console log output get alive progress periodically. Maybe some configurable time duration, say every 60s or 5m. I would probably disable the bar and spinner and just get the stats. Completion percent, count, average time per and ETA. Is this possible currently?

Thanks for the awesome library!

**Comment by rsalmei** [comment]
Thanks, man! 👍 

I'm not sure what you mean... Will this script be run for short periods of time, several times?
Or will it run nonstop for a couple of days?

And what do you mean by "would like to see the console log output get alive progress periodically"? You'd like to make it disappear somehow?
Please give a more concrete description of your case, will you?

**Comment by jrob** [comment]
It is a long running script that will execute inside a k8s cluster over several days. I would like to keep tabs on it using alive bar, but I don't want to force tty and spam the log several times per second. Waiting for stats at the end is no go and I don't want to put in place another way to estimate eta and display progress.

I would be happy manually write the stats like so:

```py
with alive_bar(len(stuff), disable=True, enrich_print=False) as bar:
    for item in list(stuff):
        # do stuff
        bar()
        # Maybe skip every 10 or 100 or something with a counter
        print(bar.stats())
```

Or tell alive bar how often to write output

```py
with alive_bar(len(stuff), refresh_rate="10m") as bar:
    for item in list(stuff):
        # do stuff
        bar()
```

**Comment by rsalmei** [comment]
I see...
Yeah, I can definitely implement this, the first option I think would be great.
Unfortunately, I'm overloaded with work atm, and learning Rust at the same time, but it is in my queue 👍

**Comment by TheTechRobo** [comment]
Rust is such a cool language  I'm very happy I tried it!

**Comment by rsalmei** [comment]
YES @TheTechRobo, it is awesome!!
It is the hardest language to learn I've ever seen, but totally worth it: modern, beautiful and fast!!
I'm working full time with it for about a year, very happy!

**Comment by TheTechRobo** [comment]
The compiler is 😍, never seen a more helpful one.  
Even detects zero-width and non-breaking spaces!

**Comment by rsalmei** [comment]
Hey @jrob, I'm glad to inform you this is done!
I'm going to release it today!!
In the end, I've chosen the second suggestion, now we can `refresh_secs=10 * 60` 👍

**Comment by jrob** [comment]
Very cool! Thanks!

**Comment by rsalmei** [comment]
You're welcome 😄

**Comment by rsalmei** [comment]
Hey, it is released! #131 
Take a look at the new readme for details! 🎉

**Comment by jrob** [comment]
I know it's not a big deal but I really appreciate that you listened and took the time to implement. Kudos!
